.flake8

```diff
@@ -0,0 +1,3 @@
+[flake8]
+ignore = E501
```

Comment:
Here's the current list of `E501` errors via Flake8: https://gist.github.com/pdehaan/27010daff64c251cb012